apps/blockchain/lib/blockchain/contract/create_contract.ex

```diff
@@ -7,28 +7,32 @@ defmodule Blockchain.Contract.CreateContract do
   alias Blockchain.Interface.{BlockInterface, AccountInterface}
   alias Block.Header
   alias Blockchain.Contract.Address
-  alias Blockchain.{Account, Contract}
+  alias Blockchain.Account
   alias EVM.{SubState, Gas}
 
-  # σ
   defstruct state: %{},
-            # s
             sender: <<>>,
-            # o
             originator: <<>>,
-            # g
             available_gas: 0,
-            # p
             gas_price: 0,
-            # v
             endowment: 0,
-            # i
             init_code: <<>>,
-            # e
             stack_depth: 0,
             block_header: nil,
             config: EVM.Configuration.Frontier.new()
 
+  @typedoc """
+  Yello Paper Terms:
+
+  - σ: state,
```

Comment:
I like this format! I think looking up terms will be easier this way and it also cleans up the hash definition. We could probably do some similar refactoring in `apps/evm/lib/evm/gas.ex` but that can be in a follow on PR

Reply:
👍  I've been trying to update any comments I find in structs to this format in the files I touch.